spack_repo/local/packages/palace/package.py

```diff
@@ -205,6 +206,9 @@ class Palace(CMakePackage, CudaPackage, ROCmPackage):
             depends_on(f"slepc{rocm_variant}", when=f"+slepc{rocm_variant}")
             depends_on(f"petsc{rocm_variant}", when=f"+slepc{rocm_variant}")
 
+    with when("+tests"):
+            depends_on(f"catch2@3")
```

Comment:
NIT

```suggestion
            depends_on("catch2@3")
```

As a clarifying comment, this implies that we wouldn't be compatible with `catch2@4` if / when that is released.

Reply:
I like the `3:` as that means that we will instantly get a CI failure once we are incompatible... Bad for getting tests to always pass, but the easiest way to catch upstream issues.